src/dbally/views/sqlalchemy_base.py

```diff
@@ -77,13 +97,17 @@ def execute(self, dry_run: bool = False) -> ViewExecutionResult:
         """
 
         results = []
-        sql = str(self._select.compile(bind=self._sqlalchemy_engine, compile_kwargs={"literal_binds": True}))
+        statement = self._select
```

Comment:
As described in the comment to `apply_aggregation` - I don't think those changes are nessesary